CODEOWNERS

```diff
@@ -58,7 +58,8 @@
 /specification/relay/ @sethmanheim @v-ajnava
 /specification/resources/ @Tiano2017 @rajshah11 @vivsriaus
 /specification/scheduler/ @pinwang81
-/specification/search/ @brjohnstmsft
+/specification/search/data-plane/ @brjohnstmsft @arv100kri
```

Comment:
can you add @bleroy as well to the data-plane owners list?

Reply:
Good catch -- will do